crates/net/network/src/transactions.rs

```diff
@@ -377,7 +377,7 @@ where
 /// This should be spawned or used as part of `tokio::select!`.
 impl<Pool> Future for TransactionsManager<Pool>
 where
-    Pool: TransactionPool + Clone + Unpin,
+    Pool: TransactionPool + Unpin + 'static,
```

Comment:
I guess it's required here due to the spawns, but doesn't need to be in the smaller traits